naplib/io/load_edf.py

```diff
@@ -89,13 +98,32 @@ def rescale(x):
             data[i*samples:(i+1)*samples] = buffer[:, ~aux_signals]
             aux_data[i*samples:(i+1)*samples] = buffer[:, aux_signals]
     
+    # Convert start_date and start_time to python objects
+    try:
+        start_date = datetime.strptime(start_date, '%m.%d.%y').date()
+    except ValueError:
```

Comment:
Would it be ok to just do 
```
except:
```
rather than excepting a specific error? Just so this doesn't raise an error just in trying to support metadata? The try: except: block below doesn't require a specific error

Reply:
The two blocks should definitely match each other. I think it's generally better to be specific about errors so internal bugs don't get discarded silently. But in this case since it's metadata, I don't see the harm in having a catch-all. I'll make it so.